src/pkg/cli/login.go

```diff
@@ -57,6 +58,21 @@ func (g OpenAuthService) login(ctx context.Context, client client.FabricClient,
 	return auth.ExchangeCodeForToken(ctx, code, tenant, 0) // no scopes = unrestricted
 }
 
+func (g OpenAuthService) loginWithDocker(ctx context.Context, fabric string, authPort int) {
+	term.Debug("Logging in to", fabric)
+
+	tenant, _ := SplitTenantHost(fabric)
+
+	go func() {
```

Comment:
As discussed: starting a Go-routine and returning is an anti-pattern. Move the Go-routine to the caller.